src/popup/components/buttons/BtnIcon.vue

```diff
@@ -101,5 +101,12 @@ export default {
       }
     }
   }
+
+  &.light {
```

Comment:
I see a problem with the BtnIcon variants - there are no validation and no list of possible variants. I suggest to create a constant similar to `SIZES` and validate the `variant` prop same way as the `size`.